catalog-rest-service/src/main/resources/json/schema/entity/data/pipeline.json

```diff
@@ -56,6 +93,29 @@
         }
       },
       "required": ["id", "name"]
+    },
+    "pipelineStatus": {
+      "description": "Series of pipeline executions, its status and task status.",
+      "type": "object",
+      "properties": {
+        "executionDate": {
```

Comment:
If a pipeline executes every 6 hours, then there will be 4 executions on the same day. 

In this case, should we change this to `executionStartTime`

Reply:
I was thinking about execution date similar to the naming in Airflow, but we could indeed change this